Containers/MediaFileStream.cs

```diff
@@ -190,46 +190,6 @@ public MediaFileStream(System.IO.FileStream stream, System.IO.FileAccess access
             m_Source = new Uri(stream.Name);
         }
 
-        [Obsolete("This constructor has been deprecated.  Please use new FileStream(SafeFileHandle handle, FileAccess access) instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
-        internal protected MediaFileStream(System.IO.FileStream stream, Uri uri, System.IO.FileAccess access = System.IO.FileAccess.Read, bool ownsHandle = true)
-            : base(stream.Handle, access, ownsHandle)
-        {
-            m_Source = uri ?? new Uri(stream.Name);
-        }
-
-        /// <summary>
-        /// Creates a new FileStream from the given.
-        /// </summary>
-        /// <param name="stream"></param>
-        /// <param name="access"></param>
-        /// <param name="uri"></param>
-        /// <param name="fi"></param>
-        /// <param name="afterClose"></param>
-        public MediaFileStream(System.IO.FileStream stream, System.IO.FileAccess access = System.IO.FileAccess.Read, Uri uri = null, System.IO.FileInfo fi = null, Action afterClose = null)
```

Comment:
Was it intentional to remove this constructor?

Reply:
Yep, it never used and calls obsolete constructor.

Reply:
If this constructor is no longer used and calls obsolete its fine to remove I just want to make sure we didn't lose any logic, is AfterClose and FileInfo assigned in other constructors? I don't see any constructor to takes this one's place :)

Reply:
This constructor is unused it seems?